modules/multy-s3-bucket/0.2.0/s3-replica-bucket.tf

```diff
@@ -33,13 +33,86 @@ resource "aws_s3_bucket_lifecycle_configuration" "replica" {
   provider   = aws.replicaregion
   depends_on = [aws_s3_bucket_versioning.replica]
   bucket     = aws_s3_bucket.replica.id
+
+  rule {
+    id = "expire_old_vault_backups"
+
+    filter {
+      prefix = "hashicorp-vault-backups/*"
+    }
+
+    expiration {
+      days = 15
```

Comment:
Can you set this to match the new policies? same rules. As these will be recent backups we want to retain

Reply:
or am i looking in the wrong spot?

Reply:
This isn't done?

Reply:
I'm still seeing a number of discprencies.